<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>GuiExit</class>
 <widget class="QDialog" name="GuiExit" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>8</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="3" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="mp_pbOpenReverseExit" >
       <property name="minimumSize" >
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>Apri uscita inversa</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>91</width>
         <height>23</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mp_pbOk" >
       <property name="text" >
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mp_pbSave" >
       <property name="text" >
        <string>Applica</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mp_pbRestore" >
       <property name="text" >
        <string>Ripristina</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mp_pbCancel" >
       <property name="text" >
        <string>Annulla</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="mp_gbAttributes" >
     <property name="minimumSize" >
      <size>
       <width>16</width>
       <height>120</height>
      </size>
     </property>
     <property name="title" >
      <string>Attributi</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>8</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="2" column="3" >
       <widget class="QLabel" name="mp_labelKey" >
        <property name="text" >
         <string>Chiave</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2" >
       <widget class="QComboBox" name="mp_comboOpenCommand" />
      </item>
      <item row="1" column="1" colspan="5" >
       <widget class="QLineEdit" name="mp_leKeys" />
      </item>
      <item row="2" column="5" >
       <widget class="QToolButton" name="mp_tbKey" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5" >
       <widget class="QToolButton" name="mp_tbFlags" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QToolButton" name="mp_tbToRoom" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="mp_labelKeys" >
        <property name="text" >
         <string>Parole chiavi</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="mp_labelFlags" >
        <property name="text" >
         <string>Flags</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="mp_leToRoom" >
        <property name="maxLength" >
         <number>6</number>
        </property>
       </widget>
      </item>
      <item row="3" column="3" >
       <widget class="QLabel" name="mp_labelInit" >
        <property name="text" >
         <string>Inizializzazione</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="4" >
       <widget class="QLineEdit" name="mp_leKey" >
        <property name="maxLength" >
         <number>6</number>
        </property>
       </widget>
      </item>
      <item row="3" column="4" colspan="2" >
       <widget class="QComboBox" name="mp_comboInit" />
      </item>
      <item row="0" column="1" colspan="4" >
       <widget class="QLineEdit" name="mp_leFlags" >
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="mp_labelToRoom" >
        <property name="text" >
         <string>Alla stanza</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="mp_labelOpenCommand" >
        <property name="text" >
         <string>Comando</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QGroupBox" name="mp_gbDescription" >
     <property name="title" >
      <string>Descrizione</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>8</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QTextEdit" name="mp_teDescription" >
        <property name="maximumSize" >
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="mp_gbMain" >
     <property name="minimumSize" >
      <size>
       <width>16</width>
       <height>100</height>
      </size>
     </property>
     <property name="title" >
      <string>Generale</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>8</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="mp_labelDirection" >
        <property name="toolTip" >
         <string>Per i comandi di movimento.</string>
        </property>
        <property name="text" >
         <string>Tipo</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="mp_comboDirection" />
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="mp_labelFromName" >
        <property name="toolTip" >
         <string>Viene usata tutte le volte che bisogna specificare da dove si proviene.</string>
        </property>
        <property name="text" >
         <string>Da</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="mp_leFromName" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="mp_labelListName" >
        <property name="toolTip" >
         <string>Il nome che compare quando c'e' una lista di uscite.</string>
        </property>
        <property name="text" >
         <string>Nome in lista</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="mp_leListName" />
      </item>
      <item row="2" column="2" >
       <widget class="QLabel" name="mp_labelRevDirName" >
        <property name="toolTip" >
         <string>E' il nome dell'uscita inversa.</string>
        </property>
        <property name="text" >
         <string>Nome uscita inversa</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="3" >
       <widget class="QLineEdit" name="mp_leRevDirName" />
      </item>
      <item row="1" column="2" >
       <widget class="QLabel" name="mp_labelToName" >
        <property name="toolTip" >
         <string>Viene usata tutte le volte che bisogna specificare un verso di direzione.</string>
        </property>
        <property name="text" >
         <string>Verso</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QLineEdit" name="mp_leToName" />
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="mp_labelName" >
        <property name="toolTip" >
         <string>Per i comandi di movimento.</string>
        </property>
        <property name="text" >
         <string>Nome (univoco)</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QLineEdit" name="mp_leName" />
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <widget class="QWidget" name="" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>402</y>
     <width>591</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="5" margin="5" />
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>mp_comboDirection</tabstop>
  <tabstop>mp_leName</tabstop>
  <tabstop>mp_leListName</tabstop>
  <tabstop>mp_leToName</tabstop>
  <tabstop>mp_leFromName</tabstop>
  <tabstop>mp_leRevDirName</tabstop>
  <tabstop>mp_tbFlags</tabstop>
  <tabstop>mp_leKeys</tabstop>
  <tabstop>mp_leToRoom</tabstop>
  <tabstop>mp_tbToRoom</tabstop>
  <tabstop>mp_leKey</tabstop>
  <tabstop>mp_tbKey</tabstop>
  <tabstop>mp_comboOpenCommand</tabstop>
  <tabstop>mp_comboInit</tabstop>
  <tabstop>mp_teDescription</tabstop>
  <tabstop>mp_pbOpenReverseExit</tabstop>
  <tabstop>mp_pbOk</tabstop>
  <tabstop>mp_pbSave</tabstop>
  <tabstop>mp_pbRestore</tabstop>
  <tabstop>mp_pbCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
